.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,15 @@
+name: tests
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - run: npm i -g yarn
+      - run: yarn install --frozen-lockfile
```

Comment:
If this is too slow we can later cache the `node_modules/` folder between runs.

Reply:
I have added a commit. Let's see if it works.

Reply:
Wow, i didn'know of this elegant way of caching 😲

Reply:
It seems to be working:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/142531/144604202-4da5753d-5514-47a6-b89c-da4dc348caec.png">

Reply:
Nice, 30 seconds shorter test cycle may add up: 
![image](https://user-images.githubusercontent.com/319826/144604389-68ed98dc-7dea-42ad-9607-64424f8ea5aa.png)
🎉